EncoreTickets.SDK/Utilities/Serializers/ISerializerWithDateFormat.cs

```diff
@@ -9,5 +9,7 @@ public interface ISerializerWithDateFormat : ISerializer, IDeserializer
         DataFormat SerializedDataFormat { get; }
 
         string DateFormat { get; set; }
+
+        T Deserialize<T>(string content);
```

Comment:
Because the standard IDeserializer interface is provided by the RestSharp library and only deserializes the RestResponse object. This method allows  deserializing JSON without thinking about the structure of RestResponse object

Reply:
But this seems unrelated to the purpose of this interface and its existing members. Are you sure we need this here and not somewhere else?

Reply:
If this should only be available for deserialization of RestResponse, maybe we need to rename this interface? And for the new method, do we need to create a separate, more general interface? Because currently it looks like a common serializer, independent of serialization / deserialization for requests / responses

Reply:
I think we definitely need to move this to a separate interface as it starts looking like an ISP violation. Also maybe we can even replace IDeserializer with this new interface, although I think that might require some work so maybe we can do this at some point in the future.